README.md

```diff
@@ -5,21 +5,6 @@ The [TTS Handbook](https://handbook.18f.gov) documents the mission, values, stru
 
 Note: If you're changing any kind of process, please let [#wg-onboarding](https://gsa-tts.slack.com/messages/wg-onboarding) know so we can keep the handbook, new hire messages, and checklists up to date.
 
```

Comment:
As we remove this, at some point in it we should probably point people to the contributing doc?

Reply:
Adding something at the bottom of the `README.md` [similar to other 18F repositories](https://github.com/18F/analytics-reporter/blob/master/README.md#public-domain).